galaxy_ng/tests/unit/migrations/test_0029_move_perms_to_roles.py

```diff
@@ -285,6 +286,39 @@ def test_group_model_locked_role_mapping_with_dangling_permissions(self):
             self.assertTrue(self._has_role(group, role))
 
 
+    def test_group_permissions_post_migration(self):
+        permissions_to_add = LOCKED_ROLES["galaxy.collection_admin"]["permissions"]
+
+        _, group = self._create_user_and_group_with_permissions("test", permissions_to_add)
+
+        # Add permissions directly to a group
+        for perm in permissions_to_add:
+            group.permissions.add(self._get_permission(perm))
+        group.save()
+
+        self.assertTrue(len(permissions_to_add) > 0)
+        self.assertEqual(group.permissions.count(), len(permissions_to_add))
+
+        self._run_migrations()
+
+        expected_role = "galaxy.collection_admin"
+        role_obj = Role.objects.get(name=expected_role)
+        self.assertEqual(GroupRole.objects.filter(group=group).count(), 1)
+        self.assertEqual(GroupRole.objects.filter(group=group, role=role_obj).count(), 1)
+        self.assertTrue(self._has_role(group, expected_role))
+
+        # check that group no longer has any permissions associated
+        group.refresh_from_db()
+        self.assertEqual(group.permissions.all().count(), 0)
```

Comment:
This PR passes with this check commented out. After #1404 this check should pass.